Example/Clappr/ViewController.swift

```diff
@@ -93,4 +89,19 @@ class ViewController: UIViewController {
         alertViewController.addAction(UIAlertAction(title: "ok", style: UIAlertAction.Style.default, handler: nil))
         self.navigationController?.present(alertViewController, animated: true, completion: nil)
     }
+    
+    private func createPlayer() {
+        player = Player(options: options)
+
+        listenToPlayerEvents()
+
+        player.attachTo(playerContainer, controller: self)
+        player.play()
```

Comment:
I think we can avoid calling a play because we are changing the behavior of the player. With this new state, the player has autoplay and we lost the Poster in the beginning. Does it make sense?

Reply:
Yes! This was just to facilitate the testing. It can be undone.